app/assets/stylesheets/student_index.scss

```diff
@@ -0,0 +1,54 @@
+@import 'color';
+
+.student_index_container {
+  margin-top: 40px;
+
+  .student_block {
+    margin-right: 10px;
+    margin-bottom: 20px;
+    height: 120px;
+    border-style: solid;
+    border-color: $rock;
+    border-width: 1px;
+
+    background-color: $gray;
+
+    .link_student a {
+      color: $rock;
+      font-weight: bold;
+      font-size: 1.4rem;
+      &:hover {
+        text-decoration: underline;
+      }
+    }
+
+    .link a {
+      color: $brick;
+      font-weight: normal;
+      font-size: 1.3rem;
+      &:hover {
+        color: lighten( $brick, 10% );
```

Comment:
Expected 0 spaces after `(` instead of ` `<br>Expected 0 spaces before `)` instead of ` `